src/pages/home/getting-started/tabs/get-the-sdk.md

```diff
@@ -41,7 +41,7 @@ For the latest Flutter installation instructions, see the package [install tab](
 Add the dependencies to `build.gradle` for each extension.
 
 ```java
-implementation platform('com.adobe.marketing.mobile:sdk-bom:2.+')
+implementation platform('com.adobe.marketing.mobile:sdk-bom:3.+')
```

Comment:
- these changes look good for android, but looks like we did not update the ios ones for 5.x. can we take those in a separate pr?
- also do we need to include kotlin installation instructions also?

Reply:
Yes. I am working on a separate PR to standardize the instructions and update iOS instructions.